spec/board_spec.rb

```diff
@@ -207,4 +207,75 @@
 
     expect(board.position_taken?(8)).to eq(true)
   end
+
+  describe 'full_board?' do
+    it 'returns true when every index in board array is full' do
```

Comment:
I wonder about mentioning `array` in the test description because it is so specific to implementation details, and we want our tests to focus on behavior. You could instead write `returns true when every position on the board is has a mark`. This is nitpicky but I'm curious about what others think.

Reply:
I was conflicted about this as well because I felt like I couldn't say "when the board is full" because what does that even mean programmatically? Also, it was redundant with the method name? I think the suggestion is valid, I can also wait for another reviewer's feedback.